<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_ejercicio</class>
 <widget class="QWidget" name="form_ejercicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Ejercicio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Liberation Sans&quot;;
color: rgb(0, 0, 0);
background-color: rgb(156, 156, 156);
selection-background-color: rgb(164, 190, 221);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ejercicio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="tw_ejercicio">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>250</number>
         </attribute>
         <column>
          <property name="text">
           <string>Año</string>
          </property>
          <property name="background">
           <color>
            <red>149</red>
            <green>175</green>
            <blue>197</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descripción</string>
          </property>
          <property name="background">
           <color>
            <red>158</red>
            <green>180</green>
            <blue>199</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
          <property name="background">
           <color>
            <red>158</red>
            <green>180</green>
            <blue>199</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="tw_detalle_ejerc">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>250</number>
           </attribute>
           <column>
            <property name="text">
             <string>N°</string>
            </property>
            <property name="background">
             <color>
              <red>158</red>
              <green>180</green>
              <blue>199</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mes</string>
            </property>
            <property name="background">
             <color>
              <red>158</red>
              <green>180</green>
              <blue>199</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
            <property name="background">
             <color>
              <red>158</red>
              <green>180</green>
              <blue>199</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cerrar">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(119, 149, 177);</string>
           </property>
           <property name="text">
            <string>Cerrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>CUIT / CUIL: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lne_cuit">
               <property name="minimumSize">
                <size>
                 <width>110</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>110</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="minimumSize">
                <size>
                 <width>85</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Razón Social: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ln_nombre">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>350</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_buscar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(119, 149, 177);</string>
               </property>
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_actualizar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(119, 149, 177);</string>
               </property>
               <property name="text">
                <string>Actualizar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_eliminar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(119, 149, 177);</string>
               </property>
               <property name="text">
                <string>Eliminar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_nuevo">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(119, 149, 177);</string>
               </property>
               <property name="text">
                <string>Nuevo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_inflacion">
         <property name="text">
          <string>calcular inflacion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lne_cuit</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>ln_nombre</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>btn_actualizar</tabstop>
  <tabstop>btn_nuevo</tabstop>
  <tabstop>tw_ejercicio</tabstop>
  <tabstop>tw_detalle_ejerc</tabstop>
  <tabstop>btn_cerrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
